fw/t/unit/test_http_parser.c

```diff
@@ -2112,138 +2387,86 @@ TEST(http_parser, set_cookie)
 			  "0123456789");
 }
 
+// For ETag and If-None-Match headers
+#define COMMON_ETAG_BLOCK(header, BLOCK_MACRO)		\
+	BLOCK_MACRO(header ": \"dummy\"\r\n"		\
+		   ": \"dummy\"");			\
+	BLOCK_MACRO(header ": \"dummy");		\
+	BLOCK_MACRO(header ": dummy\"");		\
+	BLOCK_MACRO(header ": 'dummy'");		\
+	BLOCK_MACRO(header ": W/ \"dummy\"");		\
+	BLOCK_MACRO(header ": w/\"dummy\"");		\
+	BLOCK_MACRO(header ": \"\x00\"");		\
+	BLOCK_MACRO(header ": \"\x0F\"");		\
+	BLOCK_MACRO(header ": \"\x7F\"");		\
+	BLOCK_MACRO(header ": \" \"");			\
+	BLOCK_MACRO(header ": \"\"\"")
+
 TEST(http_parser, etag)
 {
-#define RESP_ETAG_START							\
-	"HTTP/1.1 200 OK\r\n"						\
-	"Date: Mon, 23 May 2005 22:38:34 GMT\r\n"			\
-	"Content-Type: text/html; charset=UTF-8\r\n"			\
-	"Content-Encoding: UTF-8\r\n"					\
-	"Content-Length: 10\r\n"					\
-	"Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n"		\
+#define RESP_ETAG_START                                    \
+	"HTTP/1.1 200 OK\r\n"                              \
+	"Date: Mon, 23 May 2005 22:38:34 GMT\r\n"          \
+	"Content-Type: text/html; charset=UTF-8\r\n"       \
+	"Content-Encoding: UTF-8\r\n"                      \
+	"Content-Length: 10\r\n"                           \
+	"Last-Modified: Wed, 08 Jan 2003 23:11:55 GMT\r\n" \
 	"Server: Apache/1.3.3.7 (Unix) (Red-Hat/Linux)\r\n"
```

Comment:
Well, the kernel uses tabs and spaces at the end of identation for alignment. In this case, trailing macro strings, tabs should be used